packages/slate-editor/src/components/Menu/Dropdown.module.scss

```diff
@@ -52,9 +52,23 @@
 
         &:hover {
             color: $toolbar-dark-theme-text-hover-color;
+
+            &::before {
+                // Trick to display smaller hover background square,
+                // while keeping the button area as big as possible (for a11y)
+                content: "";
+                display: block;
+                position: absolute;
+                left: $spacing-base;
+                top: $spacing-base;
+                bottom: $spacing-base;
+                right: $spacing-base;
+                border-radius: $border-radius-base;
+                background-color: $toolbar-dark-theme-hover-bg;
+            }
```

Comment:
It turned out that the hover state was missing here, and instead was applied from the main app stylesheet :grimacing: